src/ship/PlayerShipController.cpp

```diff
@@ -243,6 +250,12 @@ void PlayerShipController::InputBinding::RegisterBindings()
 
 	toggleLandingGear = AddAction("BindToggleLandingGear");
 	controlLandingGear = AddAxis("BindControlLandingGear");
+
+	AddAction("BindRadarToggleMode");
+	AddAction("BindRadarZoomReset");
+	// TODO: Convert to axis?
```

Comment:
As mentioned above, I'd recommend adding an additional axis which is only activated when hovering the radar widget. Future code could also make the axis "active" when the radar widget has received some sort of input focus, for use with e.g. controllers.

Reply:
Axis partially added in d0e849130043b3ccf6a84d15cc0c026a8e5e542c ; will need to investigate how this can be activated on mouse-over.

Reply:
Axis removed again until I can fully implement it.